keras_cv/layers/preprocessing/random_gaussian_blur.py

```diff
@@ -33,10 +33,12 @@ class RandomGaussianBlur(BaseImageAugmentationLayer):
         factor: A tuple of two floats, a single float or a
             `keras_cv.FactorSampler`. `factor` controls the extent to which the
             image is blurred. Mathematically, `factor` represents the `sigma`
-            value in a gaussian blur. `factor=0.0` makes this layer perform a
-            no-op operation, and high values make the blur stronger. In order to
-            ensure the value is always the same, please pass a tuple with two
-            identical floats: `(0.5, 0.5)`.
+            value in a gaussian blur. The closer `factor` is to 0, the more 
```

Comment:
It looks like we can just fix this numerical issue by adding an epsilon to `factor`  on line 115 where we divide by `factor` -- this should resolve the issue.

Reply:
In my code I now use 0.01 and it gives the intended behavior. 
```
>>> blurrer = keras_cv.layers.RandomGaussianBlur(3, 0.01)
>>> blurrer.get_random_transformation()
(<tf.Tensor: shape=(3, 1, 1, 1), dtype=float32, numpy=
array([[[[0.]]],


       [[[1.]]],


       [[[0.]]]], dtype=float32)>, <tf.Tensor: shape=(1, 3, 1, 1), dtype=float32, numpy=
array([[[[0.]],

        [[1.]],

        [[0.]]]], dtype=float32)>)
```